core/src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -62,45 +65,63 @@ object ManagedGroup {
-                               googleProjectNumber: Option[GoogleProjectNumber] = None,
-                               invalidBillingAccount: Boolean = false,
-                               spendReportDataset: Option[BigQueryDatasetName] = None,
-                               spendReportTable: Option[BigQueryTableName] = None,
-                               spendReportDatasetGoogleProject: Option[GoogleProject] = None,
-                               azureManagedAppCoordinates: Option[AzureManagedAppCoordinates] = None,
-                               billingProfileId: Option[String] = None
+case class RawlsBillingProject(
+  projectName: RawlsBillingProjectName,
+  status: CreationStatuses.CreationStatus,
+  billingAccount: Option[RawlsBillingAccountName],
+  message: Option[String],
+  cromwellBackend: Option[CromwellBackend] = None,
+  servicePerimeter: Option[ServicePerimeterName] = None,
+  googleProjectNumber: Option[GoogleProjectNumber] = None,
+  invalidBillingAccount: Boolean = false,
+  spendReportDataset: Option[BigQueryDatasetName] = None,
+  spendReportTable: Option[BigQueryTableName] = None,
+  spendReportDatasetGoogleProject: Option[GoogleProject] = None,
+  azureManagedAppCoordinates: Option[AzureManagedAppCoordinates] = None,
+  billingProfileId: Option[String] = None
 ) {
   // def instead of val because val confuses the json formatter
   def googleProjectId: GoogleProjectId = GoogleProjectId(projectName.value)
 }
 
-case class WorkspaceBillingAccount(workspaceName: WorkspaceName,
-                                   currentBillingAccountOnGoogleProject: Option[RawlsBillingAccountName]
+case class WorkspaceBillingAccount(
+  workspaceName: WorkspaceName,
+  currentBillingAccountOnGoogleProject: Option[RawlsBillingAccountName]
 )
 
-case class RawlsBillingProjectResponse(projectName: RawlsBillingProjectName,
-                                       billingAccount: Option[RawlsBillingAccountName],
-                                       servicePerimeter: Option[ServicePerimeterName],
-                                       invalidBillingAccount: Boolean,
-                                       roles: Set[ProjectRoles.ProjectRole],
-                                       status: CreationStatuses.CreationStatus,
-                                       message: Option[String],
-                                       managedAppCoordinates: Option[AzureManagedAppCoordinates]
+case class RawlsBillingProjectResponse(
+  projectName: RawlsBillingProjectName,
+  billingAccount: Option[RawlsBillingAccountName],
+  servicePerimeter: Option[ServicePerimeterName],
+  invalidBillingAccount: Boolean,
+  roles: Set[ProjectRoles.ProjectRole],
+  status: CreationStatuses.CreationStatus,
+  message: Option[String],
+  managedAppCoordinates: Option[AzureManagedAppCoordinates], // remove after ui is updated  to use cloud context
+  cloudPlatform: String
 )
 
+object RawlsBillingProjectResponse {
```

Comment:
Using an apply method like this seemed a lot more clear than using UserAuth. makeBillingProjectResponse.

I can remove it if anyone disagrees, or switch all the usages of makeBillingProjectResponse to this, if we want.

Reply:
I don't think we want both methods, though I don't have a preference about which one you keep.

Reply:
Currently this method is only used for the implicit parsing of json?

Reply:
And why does it need to be `apply` there vs. just `RawlsBillingProjectResponse`?

https://github.com/broadinstitute/rawls/pull/2010/files#diff-a257c53a0296aad7d58c01a06ae5a075382008742e4fc3bca818a0d172705814R307

Reply:
The short answer is to conform to other usages of the class.

The longer answer has to do with the details and purpose of case classes:

The idea of a case class is that it is to represent immutable data in a concise way, provide some conveniences around it automatically, and allow it to be easily de-structured in match statements.

When you define something as a case class, one thing the compiler does is define a default apply method with the class arguments. That's why you can just use `RawlsBillingProjectResponse(... args)` instead of the `new` keyword. 

What we're doing here is essentially overloading the `constructor`, but since it's a case class it makes for more consistent syntax to use `apply`, so you don't have to use a mix of `new` and the apply method (which would be _really_ confusing).